aospy/model.py

```diff
@@ -72,14 +86,43 @@ def find_data_in_direc_repo(self, run_name='amip', var_name='ta',
 
         return glob.glob(os.path.join(direc_full, var_name + '_*.nc'))
 
+    def _get_grid_files_repo(self):
```

Comment:
I think we should eventually abstract this in a similar way that we are planning on abstracting the variable retrieval process (#32).  Right now it's a little too GFDL / CMIP5 specific (in terms of hard-coded directory and variable names) to reside in the aospy core.